llvm/lib/Target/DirectX/DXILShaderFlags.h

```diff
@@ -95,6 +95,8 @@ struct ModuleShaderFlags {
   SmallVector<std::pair<Function const *, ComputedShaderFlags>> FunctionFlags;
   /// Combined Shader Flag Mask of all functions of the module
   ComputedShaderFlags CombinedSFMask{};
+  auto getFunctionShaderFlagInfo(const Function *) const;
```

Comment:
LLVM has an "almost never `auto`" policy. We should only use `auto` where it makes code easier to read and where types are obvious from other context. That is not the case here.

see: https://llvm.org/docs/CodingStandards.html#use-auto-type-deduction-to-make-code-more-readable

Reply:
Function is deleted in commit that addresses the PR feedback..